src/crud/workspace.py

```
@@ -241,10 +292,17 @@ async def delete_workspace(db: AsyncSession, workspace_name: str) -> schemas.Wor
         )
         await db.delete(honcho_workspace)
         await db.commit()
+        print(f"Evicting workspace {workspace_name} from cache")
+        await _workspace_cache.evict_resource(
+            workspace_cache_key(workspace_name),
+        )
         logger.debug("Workspace %s deleted", workspace_name)
     except Exception as e:
         logger.error("Failed to delete workspace %s: %s", workspace_name, e)
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)